src/main/java/core/basesyntax/BetweenDates.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class BetweenDates {
+    private final SimpleDateFormat formatter = new SimpleDateFormat("dd.MM.yyyy");
+    private Date dateFrom;
```

Comment:
[Use LocalDate instead of Date](https://github.com/mate-academy/jv-salary-info/blob/master/checklist.md#use-localdate-instead-of-date)